src/state/hooks.ts

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react'
+import { kebabCase } from 'lodash'
+import { Toast, toastTypes } from '@pancakeswap-libs/uikit'
+import { useDispatch } from 'react-redux'
+
+import {
+  push as pushToast,
+  remove as removeToast,
+  clear as clearToast
+} from './actions'
+
+// Toasts
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why do you disable , instead of using `export default` ?

Reply:
done